tailwind.config.js

```diff
@@ -8,6 +8,34 @@ module.exports = {
   },
   presets: [require('./packages/client/tailwind.preset.js')],
   content: ['./packages/client/**/*.{ts,tsx,js,jsx,html}', './template.html'],
+  safelist: [
+    'bg-aqua-400',
```

Comment:
This lets us use these colours in a variable in tailwind. In the parent issue, click on "Retrospective" and see that the background colour doesn't show up in the button, but it does in this PR.

![Screenshot 2023-11-27 at 16 32 49](https://github.com/ParabolInc/parabol/assets/39854876/782f2a31-e8ce-4db9-93d0-9eb804bf01c2)

Reply:
In the previous PR you mentioned [here](https://github.com/ParabolInc/parabol/pull/9168#discussion_r1396147758) that you found it confusing to have both text & bg colour variables. 

Would you like to revert back to that or something else? Happy for me to make that change in a separate PR?

Reply:
@Dschoordsch can we merge this PR without updating these colours? I could do a quick release with this, gcal, and the checkout flow

Reply:
Let's just move all the colours into the theme, add a third field `{primary: 'bg-...', secondary:..., text: 'text-...'}` and remove the safelist. That shouldn't take long at all.

Reply:
I've updated the `CATEGORY_THEMES` object adding a third `text` key